galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -348,3 +348,14 @@ messages:
       - name: err
         type: error
 
+  - name: "ServiceEntryMissingAddressesAndProtocol"
```

Comment:
nit: Not sure we need two different messages for these - they are essentially identical